deployment/helm/resource-management-policies/balloons/templates/daemonset.yaml

```
@@ -78,3 +91,13 @@ spec:
         hostPath:
           path: /var/run/nri
           type: Directory
+      {{- if .Values.nri.patchContainerdConfig }}
+      - name: containerd-config
+        hostPath:
+          path: /etc/containerd/config.toml
+          type: File
+      - name: dbus-sockets
+        hostPath:
+          path: /var/run/dbus
+          type: Directory
```

nit: why not mount the file that we really need? This would verify its existence (and type) on the host
```suggestion
          path: /run/dbus/system_bus_socket
          type: Socket
```